src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.csproj

```diff
@@ -82,6 +82,10 @@
       <Project>{47533B7C-0E1A-44A4-8511-B438645F052A}</Project>
       <Name>DynamoPackages</Name>
     </ProjectReference>
+    <ProjectReference Include="..\PythonMigrationViewExtension\PythonMigrationViewExtension.csproj">
+      <Project>{10af430d-0d3a-49ce-a63d-848912959745}</Project>
```

Comment:
yuck!

Reply:
We can discuss.. Any other work around you guys can see @DynamoDS/dynamo? @aparajit-pratap you mentioned moving `ContainsIronPythonDependencies` to DynamoCore does not help correct?

Reply:
This function has a dependency on PythonNodeModels and DynamoCore doesn't.